src/css/controls/imports/controlbar.less

```diff
@@ -82,13 +82,23 @@
 }
 
 .jw-button-container {
+    display: block;
     display: flex;
     flex-flow: row nowrap;
     flex: 1 1 auto;
     align-items: center;
     justify-content: center;
     width: 100%;
     padding: 0 12px;
+
+    div {
+        float: left;
+        line-height: 21px;
+    }
+
+    .jw-spacer ~ div {
+        float: right;
```

Comment:
The control bar buttons on the right will show up in reverse order in old Webkit browsers missing flex-box support, but that's better than having them jammed up against the buttons on the left.